### DISCUSSION TITLE
CANNOT GENERATE IMAGE IN FOOOCUS.

### DISCUSSION THREAD

**Discussion by YoiRida** [lllyasviel/Fooocus#2276] [Q&A] [answered]
anyone help me. i cant generate image in fooocus
my pc : amdR54600
gpu rx580

![image](https://github.com/lllyasviel/Fooocus/assets/157958018/a0d94276-1a19-4a43-a83a-795774c87b6c)

**Comment by poor7** [comment]
What does it say in the log?

**Comment by YoiRida** [comment]
raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled . in cmd

log is empty

 {"detail":"File not found: H:/Instalan Game/AI Stable Defusion/Fooocus_win64_2-1-831/Fooocus/outputs/2024-02-17/log.html."}

**Comment by ItsJoshBrown** [comment, marked as answer]
> If you've ever ran `pip install torch` without the `-f https://download.pytorch.org/whl/torch_stable.html` argument, chances are, you have the corrupt cpu only version in your pip cache. whatever reinstall you do, doesn't matter because pip will just pull the same bad version from cache. to solve:
> 
> ```
> pip uninstall torch
> pip cache purge
> pip install torch -f https://download.pytorch.org/whl/torch_stable.html
> ```

Hopefully, this helps if you are having the same issue. A couple of people seemed to have a similar issue and said that this was able to resolve it for them. 
https://github.com/pytorch/pytorch/issues/30664#issuecomment-757431613
